apps/meteor/app/otr/client/OTR.ts

```diff
@@ -24,6 +24,10 @@ class OTR implements IOTR {
 		this.instancesByRoomId[rid] = otrRoom;
 		return this.instancesByRoomId[rid];
 	}
+
+	closeAllInstances(): void {
```

Comment:
During closing all instances of OTR, we should also call the cleanup functions of each of the OTR-rooms, in this case calling the `end` function for each room.

Reply:
Yeah, was calling .end here first but problem is that at some point end calls removeoldotrmessages which is an auth call but since onLogout happens after, theres no auth.

Wdyt of changing those hooks to just to internal cleaning? Like reset but with the rest of the calls from end.